### ISSUE TITLE
Publish rom-rethinkdb to RubyGems

### ISSUE THREAD

**Issue by angeloashmore** [rom-rb/rom#300]
Are there plans to publish [rom-rb/rom-rethinkdb](https://github.com/rom-rb/rom-rethinkdb) to RubyGems? Or is it still a WIP and not ready for use?

**Comment by solnic** [comment]
Yes, unfortunately nobody has found the time to do that yet :) I can push what we have as 0.0.1. It probably needs a couple smaller updates but it shouldn't take too long.

Are you a RethinkDB user? I've been reading about it and it seems pretty awesome. Would love to try it out.

**Comment by angeloashmore** [comment]
I used RethinkDB a bit several months back just to try it out. It's a cool and definitely well thought out system. I'm looking at using it for a new project where realtime streams would be beneficial.

I just converted some of my codebase from rom-sql to rom-rethinkdb (it's amazing how easy it is with ROM) and was met with some limitations in the rom-rethinkdb gem. I forked it so I'll try to add some functionality.
